### PR TITLE
✨ Update amp-access-scroll

### PR DESCRIPTION
**Pull request by junoatscroll**
This PR
- adds a protocol version query param to requests made from amp-access-scroll for compatibility
- adds a "holdback" flag and styles to prevent mismatched rendeirng with new layout
- renames ScrollAudio to ScrollSheet
- adds ability to dynamically size horizontal layout for ScrollComponents. Special care
  was taken to ensure these properties cannot interrupt vertical browser layout;
  only width, left, and right are allowed.
  Dynamic sizing is necessary to pervent invisible iframes being drawn over usable
  parts of the underlying page, which would cause UX and accessibility problems in
  certain layout scenarios. Dynamic sizing solves this by shrinking the iframe to fit
  the size neccessary to fit the rendered controls.

### CODE REVIEW

File: extensions/amp-access-scroll/0.1/amp-access-scroll.css [hunk 1]
```diff
@@ -15,34 +15,75 @@
...
-  background-color: transparent;
+.amp-access-scroll-bar.amp-access-scroll-holdback {
+  height: 44px;
+  left: 0;
+  right: 0;
+  width: 100%;
+}
+
+.amp-access-scroll-sheet.amp-access-scroll-holdback {
   position: fixed;
```
Comment (dbow) [inline extensions/amp-access-scroll/0.1/amp-access-scroll.css]: seems like these styles are unnecessary with the `.amp-access-scroll-sheet` classname providing them too?

File: extensions/amp-access-scroll/0.1/amp-access-scroll.css [hunk 2]
```diff
@@ -15,34 +15,75 @@
...
+  .amp-access-scroll-sheet.amp-access-scroll-holdback {
     bottom: 63px;
     height: 56px;
     left: auto;
     right: 16px;
     width: 475px;
   }
 }
+
+.amp-access-scroll-sheet {
```
Comment (dbow) [inline extensions/amp-access-scroll/0.1/amp-access-scroll.css]: could you put this classname above the modifications to it?

Comment (junoatscroll) [reply extensions/amp-access-scroll/0.1/amp-access-scroll.css]: Done.

File: extensions/amp-access-scroll/0.1/amp-access-scroll.css (hunk 1 above)
Comment (junoatscroll) [reply extensions/amp-access-scroll/0.1/amp-access-scroll.css]: Done.

**Comment by junoatscroll** [comment]
To clarify, the holdback code / css classes will be removed in a future PR.

**Comment by dvoytenko** [comment]
LGTM from me. Ready to be merged?

**Comment by junoatscroll** [comment]
@dvoytenko yep, thanks!

**Comment by kushal** [comment]
Yup!

**Pull request merged**